Check the validity of a CSV file.

HeartPattern opened the pull request:
Check the existence of students.csv file before reading it.

sangkilc commented on src/Server/DB.fs:
```diff
@@ -44,6 +44,10 @@ let private parseLine (line: string) =
 
+let assertDBExistence dbpath =
+  if File.Exists dbpath then ()
+  else failwith "Cannot reload: DB doesn't exist."
```
Now that this function is used in another context, the message is wrong.

sangkilc replied:
This function should not be moved since it is not called by initDBDir.

sangkilc commented on src/Server/DB.fs:
```diff
@@ -54,8 +58,9 @@ let private initSessionDir stime =
-  Path.Combine (dbdir, studentfile)
-  |> File.ReadLines
+  let dir = Path.Combine (dbdir, studentfile)
+  assertDBExistence dir
```
Although there's no big difference in this particular case, showing a well-formed error message is a good practice.

sangkilc reviewed (changes requested):
Your final modification should be only about Line 61-64.

HeartPattern committed:
Check the validity of a CSV file.

Check the existence of students.csv file before reading it.

Print well-formed error message

Print well-formed error message in validating existence of students.csv.

Revert "Check the validity of a CSV file."

This reverts commit 0ca02822a2e92ebd1b414c9fe6f101b04cc149d7.

sangkilc commented:
Thanks for your contribution. What's your name?

HeartPattern commented:
My name is 홍연준 (student id 20190698)
